### DISCUSSION TITLE
transection fee

### DISCUSSION THREAD

**Discussion by shubhampra156565** [hiero-ledger/hiero-improvement-proposals#629] [Q&A]
is there any way such that for a perticular tokken trasection fee is deducted from a single account, or it is possible to pay trasectin fee in other currency

**Comment by mgarbs** [comment]
> is there any way such that for a perticular tokken trasection fee is deducted from a single account, or it is possible to pay trasectin fee in other currency

Hey @shubhampra156565, nice to see you here. This article should help you https://hedera.com/blog/how-to-exempt-hedera-accounts-from-custom-token-fees

Additionally, check out our Discord server, you may find more answers there than here. If you don't find it there, a discussion should happen whether or not your feature would be good for the network and then generally you'll migrate that discussion here. Thank you!
[Discord](https://hedera.com/discord)
